requirements/requirements.pali.local.txt

```diff
@@ -1,7 +1,8 @@
-transformers>=4.41.1
+transformers==4.42.2
```

Comment:
Moving from a >= to == here; intentional pin? This may cause heartache if people (or we) are already using transformers elsewhere for other models.

Currently also have a >= in [requirements.cogvlm.txt](https://github.com/roboflow/inference/blob/3631d0e457d3b4701e62fe5fd266632a287d3fad/requirements/requirements.cogvlm.txt#L3) and [requirements.groundingdino.txt](https://github.com/roboflow/inference/blob/3631d0e457d3b4701e62fe5fd266632a287d3fad/requirements/requirements.groundingdino.txt#L2)

Reply:
Let me double check that when I'm back